src/widgets/allergies/allergy-form.component.tsx

```diff
@@ -299,6 +299,7 @@ export default function AllergyForm(props: AllergyFormProps) {
                   name="allergenType"
                   value={ENVIROMENTAL_ALLERGEN_CONCEPT}
                   onChange={handleAllergenChange}
+                  data-testid="environmentalAllergy"
                 />
                 <span id="ENVIROMENTAL">Enviromental</span>
```

Comment:
Shouldn't that id be also `ENVIRONMENT`, just for consistency?

Reply:
Yes, I can change the test id to `ENVIRONMENT` for maintaining the consistency. In other tests, kebab case is used for test ids. Which one should I use as test id `ENVIRONMENT` or `environment-allergen-concept`?

Reply:
If its really only about styling I would agree yes. Sounds good!

(I haven't checked where the ID / element is used)